### DISCUSSION TITLE
Draw random enclosed shape using lines or curves

### DISCUSSION THREAD

**Discussion by julurisaichandu** [pmndrs/drei#1284] [Q&A]
Hi @drcmda,
Is there any way that we can draw random enclosed shape using multiple line segments or curves? I would like to implement it just like its in the paint tool.
The other thing is that, i would also like to select all the objects which are inside the region enclosed by the drawing

**Comment by julurisaichandu** [comment]


**Reply by hmans** [reply]
Please do not ping developers.
